### ISSUE TITLE
Issue with opening the program 

### ISSUE THREAD

**Issue by xDanSenpai** [B16f00t/whapa#58]
Yesterday (22/06/2020) I got it working fine, however today (23/06/2020) when I tried to open the "whapa-gui.py" It brings up black cmd box for half a second and then it goes away but doesn't open the program. The cmd box doesn't have any writing to indicate errors. I tried to open it via the Command prompt and the same thing happens. 

I also tried to used the "whapa-gui.bat" but that does the same, pops up with the black cmd box with writting (the writing only states the file path , e.i "downloads>whapa-master>python whapa.gui.py) 

Now when I tried to execute the "whapa-gui.bat" via the CMD line, instead of clicking on the .bat file, I get this error 

**Traceback (most recent call last):
  File "whapa-gui.py", line 1049, in <module>
    Whapa(img_folder, icons)
  File "whapa-gui.py", line 521, in __init__
    update = (request.text.split('itemprop="about">')[1]).split("</span>")[0].strip("\n ")
IndexError: list index out of range**

Basically I've deleted and reinstalled python a few times, made sure everything was installed properly, everything was updated and was getting no errors, everything was coming back perfectly fine and up to date with the latest version and PIP3 updated.

Also made sure the PATH was set properly and pointed towards the right place. 

As I said, last night it was working perfectly fine, but today itt's just like the Whapa-gui doesn't appear and it's a shame because it was working like a charm last night. 

If you need anymore information or anything else, just reply and I will get back to you Asap.

**Comment by yurone-crypto** [comment]
Same problem, any solution?

**Comment by XenonKusanagi** [comment]
Same here

**Comment by nitishrajsingh** [comment]
Same here

**Comment by B16f00t** [comment]
Ok it is a simple issue, in a few hours I'll release the fix version

**Comment by yurone-crypto** [comment]
Thankyou!!

**Comment by xDanSenpai** [comment]
Awesome news, thank you so much. 

Can you drop a message once it's done please. 

Also thank you for this amazing program, honestly a work of pure art :D !

**Comment by B16f00t** [comment]
#57

**Comment by xDanSenpai** [comment]
Awesome and fast working. 

It works even smoother than it was when I intialy used it. 

Thanks for all your hard work
